attachments.md

```diff
@@ -42,3 +42,18 @@ The following table provides examples of file names for cover art in Attachments
 | small_cover.png      | Portrait or square | 120                           |
 | cover_land.png       | Landscape          | 600                           |
 | small_cover_land.jpg | Landscape          | 120                           |
+
+## Font files
+
+Font files **MAY** be added to a Matroska file as Attachments so that the font file may be used
+to display an associated subtitle track. This allows the presentation of a Matroska file to be
+consistent in various environments where the needed fonts might not be available on the local system.
+
+To associate a font file with a subtitle track, the font file **MUST** be stored as an Attachment
+within the same Segment.
```

Comment:
This sounds like subtitles cannot be used if the font is not found in the file. But if the font is not found, it's up to the subtitle codec to decide what to use. I would replace the sentence with a "\*\*MAY\*\*".